src/labels/CustomLabels.labels

```diff
@@ -7530,15 +7530,15 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <protected>true</protected>
         <shortDescription>Failure msg description for system error &amp; uncertain if card charged - Part 1</shortDescription>
         <!-- {0} = Donation Amount {1} = Donor Name {2} = custom label commonPaymentServices i.e. Payment Services -->
-        <value>A system error occurred for the {0} donation by {1}. Your admin should review transactions in {2} and determine if the card was charged.</value>
+        <value>A system error occurred for the {0} donation by {1}. Your admin should review transactions in {2} and determine if the payment was processed.</value>
     </labels>
     <labels>
         <fullName>geErrorUncertainCardChargePart2</fullName>
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Failure msg description for system error &amp; uncertain if card charged - Part 2</shortDescription>
-        <value>If the card was not charged, reenter the gift and card payment in Gift Entry.</value>
+        <value>- If the payment was not processed, reenter the gift and payment information in Gift Entry.</value>
```

Comment:
```suggestion
        <value>If the payment was not processed, reenter the gift and payment information in Gift Entry.</value>
```

Reply:
@dfuller114 couple of things:

- The "-" represents a bullet. I'll remove it from the labels, but pleases make sure the errors display as a bulleted list.
- Do we need to generalise this error to be relevant for entering Elevate payments in Recurring Donations?

Reply:
@melissabarber yeah we are displaying the label in an `<li>` element in the component markup